<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>734</width>
    <height>511</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Transient Absorption DAQ</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(50, 50, 50);
color: rgb(255, 255, 255);
font: 10pt &quot;Comic Sans MS&quot;;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QFrame" name="frame">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <widget class="QGroupBox" name="groupBox">
        <property name="title">
         <string>Stage</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_6">
         <item>
          <widget class="QPushButton" name="pushButton">
           <property name="maximumSize">
            <size>
             <width>179</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="focusPolicy">
            <enum>Qt::TabFocus</enum>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgb(30, 30, 30);</string>
           </property>
           <property name="text">
            <string>PushButton</string>
           </property>
           <property name="default">
            <bool>true</bool>
           </property>
           <property name="flat">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="spinBox">
           <property name="maximumSize">
            <size>
             <width>179</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgb(30, 30, 30);</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="comboBox">
           <property name="maximumSize">
            <size>
             <width>179</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgb(30, 30, 30);</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer_2">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QGroupBox" name="groupBox_2">
        <property name="title">
         <string>Camera</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_7">
         <item>
          <widget class="QPushButton" name="btn_getSingleAcq">
           <property name="maximumSize">
            <size>
             <width>179</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="focusPolicy">
            <enum>Qt::TabFocus</enum>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgb(30, 30, 30);</string>
           </property>
           <property name="text">
            <string>Single Acquisition</string>
           </property>
           <property name="checkable">
            <bool>false</bool>
           </property>
           <property name="autoDefault">
            <bool>false</bool>
           </property>
           <property name="default">
            <bool>true</bool>
           </property>
           <property name="flat">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btn_contAcq">
           <property name="maximumSize">
            <size>
             <width>179</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="focusPolicy">
            <enum>Qt::TabFocus</enum>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgb(30, 30, 30);</string>
           </property>
           <property name="text">
            <string>Continuous Acquisition</string>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="default">
            <bool>true</bool>
           </property>
           <property name="flat">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <item>
            <widget class="QSlider" name="sldr_absScale">
             <property name="maximum">
              <number>10000</number>
             </property>
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="tickPosition">
              <enum>QSlider::TicksAbove</enum>
             </property>
             <property name="tickInterval">
              <number>1000</number>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSlider" name="sldr_diffScale">
             <property name="maximum">
              <number>2000</number>
             </property>
             <property name="singleStep">
              <number>1</number>
             </property>
             <property name="pageStep">
              <number>10</number>
             </property>
             <property name="value">
              <number>5</number>
             </property>
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="tickPosition">
              <enum>QSlider::TicksAbove</enum>
             </property>
             <property name="tickInterval">
              <number>50</number>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_8">
      <item>
       <widget class="QGroupBox" name="groupBox_3">
        <property name="title">
         <string>Plots</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QWidget" name="mplwindow" native="true">
           <property name="autoFillBackground">
            <bool>false</bool>
           </property>
           <layout class="QVBoxLayout" name="mplvl">
            <item>
             <spacer name="horizontalSpacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QWidget" name="mplwindow2d" native="true">
           <layout class="QVBoxLayout" name="mplvl2d"/>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_10">
      <item>
       <widget class="QGroupBox" name="groupBox_4">
        <property name="maximumSize">
         <size>
          <width>201</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="title">
         <string>Auxiliary info</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QToolBox" name="toolBox">
           <property name="maximumSize">
            <size>
             <width>181</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="frameShape">
            <enum>QFrame::NoFrame</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Plain</enum>
           </property>
           <property name="currentIndex">
            <number>1</number>
           </property>
           <widget class="QWidget" name="page_1">
            <property name="geometry">
             <rect>
              <x>0</x>
              <y>0</y>
              <width>181</width>
              <height>254</height>
             </rect>
            </property>
            <attribute name="label">
             <string>How to</string>
            </attribute>
            <layout class="QVBoxLayout" name="verticalLayout_11">
             <item>
              <widget class="QPlainTextEdit" name="plainTextEdit">
               <property name="styleSheet">
                <string notr="true">background-color: rgb(255, 255, 255);</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
           <widget class="QWidget" name="page">
            <attribute name="label">
             <string>Bla bla</string>
            </attribute>
           </widget>
           <widget class="QWidget" name="page_2">
            <property name="geometry">
             <rect>
              <x>0</x>
              <y>0</y>
              <width>181</width>
              <height>254</height>
             </rect>
            </property>
            <attribute name="label">
             <string>measurement info</string>
            </attribute>
            <layout class="QVBoxLayout" name="verticalLayout_12">
             <item>
              <widget class="QPlainTextEdit" name="plainTextEdit_2">
               <property name="styleSheet">
                <string notr="true">background-color: rgb(255, 255, 255);</string>
               </property>
               <property name="frameShape">
                <enum>QFrame::NoFrame</enum>
               </property>
               <property name="frameShadow">
                <enum>QFrame::Plain</enum>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_2">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>50</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Comic Sans MS</family>
          <pointsize>10</pointsize>
          <weight>50</weight>
          <italic>false</italic>
          <bold>false</bold>
         </font>
        </property>
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 0, 0);</string>
        </property>
        <property name="text">
         <string>STOP</string>
        </property>
        <property name="autoDefault">
         <bool>false</bool>
        </property>
        <property name="default">
         <bool>false</bool>
        </property>
        <property name="flat">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>734</width>
     <height>24</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="action_Exit"/>
   </widget>
   <addaction name="menu_File"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action_Exit">
   <property name="text">
    <string>&amp;Exit</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
